src/app/components/core/notifications/add-notification-wizard/add-notification-wizard.component.html

```diff
@@ -196,7 +196,20 @@
           </div>
           <div class="field is-horizontal">
             <div class="field-label">
-              <label class="label has-text-left tooltip is-tooltip-right is-tooltip-multiline" data-tooltip="Enable this rule. Note that for debugging, a rule can be enabled while the notification is disabled.">Enabled</label>
+              <label class="label has-text-left tooltip is-tooltip-right is-tooltip-multiline" data-tooltip="Retrigger time in seconds for sending a new notification.">Retrigger Time</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control">
+                  <input class="input is-small retrigger-time" #retriggerTime type='number' appNumberOnly min="0" placeholder="Retrigger Time"
```

Comment:
put default value 60 and add client side validation validation if negative value